src/System.Linq/src/System/Linq/Concat.cs

```diff
@@ -114,7 +114,7 @@ private sealed partial class ConcatNIterator<TSource> : ConcatIterator<TSource>
             /// </summary>
             /// <remarks>
             /// This flag allows us to determine in O(1) time whether we can preallocate for <see cref="ToArray"/>
-            /// and <see cref="ConcatIterator{TSource}.ToList"/>, and whether we can get the count of the iterator cheaply.
+            /// and <see cref="M:ConcatIterator{TSource}.ToList"/>, and whether we can get the count of the iterator cheaply.
```

Comment:
❕ This is not the correct form for this method reference. To see the correct form, you can use the following approach:

1. Restore the `cref` value to its original form
2. Compile the assembly for a target where `ConcatIterator<TSource>` is present
3. Open the output XML file and find the documentation for this member
4. Copy the qualified reference (starts with `M:`) to this location

This will produce a valid reference during UAP builds as well as non-UAP builds.